### DISCUSSION TITLE
Integrate with Gitlab

### DISCUSSION THREAD

**Discussion by Aerya** [plausible/analytics#2488] [Anything Else]
Hi,

Any advice on how to integrate with Gitlab? (docker hosted one)
I'm behind an nginx reverse proxy.

Thanks

**Comment by metmarkosaric** [comment]
hi @Aerya! as in to track visitors of Gitlab? does Gitlab allow you to insert scripts into the header section? in that case, it should work like any other site

**Comment by Aerya** [comment]
Hi, yeah that's it. I also aked on Gitlabs forums. I'll post here when I got a solution.
